app/models/item.rb

```diff
@@ -3,7 +3,13 @@ class Item < ActiveRecord::Base
 
   validates_presence_of :name, :price, :quantity, :shelf_life_days
 
-  def expires
-    Date.today + shelf_life_days.days
+  scope :valid_items, -> {where 'shelf_life_days > (Date.today - created_at).in_days'}
+
```

Comment:
Yep.  That string is a sql query.  So right now this is turning into this query:
```
SELECT *
FROM items
WHERE shelf_life_days > (Date.today - create_at).in_days
```
This is going to be a little trickier than I think brandon suspected because of that calculation.